packages/browsers/src/CLI.ts

```diff
@@ -77,27 +78,53 @@ export class CLI {
       type: 'string',
       desc: 'Platform that the binary needs to be compatible with.',
       choices: Object.values(BrowserPlatform),
-      defaultDescription: 'Auto-detected by default.',
+      defaultDescription: 'Auto-detected',
     });
   }
 
   #definePathParameter(yargs: yargs.Argv<unknown>): void {
     yargs.option('path', {
       type: 'string',
-      desc: 'Path to the root folder for the browser downloads and installation',
+      desc: 'Path to the root folder for the browser downloads and installation. The installation folder structure is compatible with the cache structure used by Puppeteer.',
       default: process.cwd(),
+      defaultDescription: 'Current working directory',
     });
   }
 
   async run(argv: string[]): Promise<void> {
     await yargs(hideBin(argv))
+      .scriptName('@puppeteer/browsers')
       .command(
         'install <browser>',
-        'Download and install the specified browser',
+        'Download and install the specified browser. If successful, the command outputs the actual browser buildId that was installed and the absolute path to the browser executable.',
```

Comment:
What do you think about explicitly telling what the pattern of the line is `<browser>@<buildID>  <path>` ?

Reply:
Done